cmd/heptio-authenticator-aws/token.go

```diff
@@ -35,7 +35,7 @@ var tokenCmd = &cobra.Command{
 		clusterID := viper.GetString("clusterID")
 
 		if clusterID == "" {
-			fmt.Fprintf(os.Stderr, "error: cluster ID not specified\n")
+			fmt.Fprintf(os.Stderr, "Error: cluster ID not specified\n")
```

Comment:
Updated to be cohesive with the default Cobra error responses. `heptio-authenticator-aws token -i` for example returns:

```sh
Error: flag needs an argument: 'i' in -i
```